src/test/java/com/scriptbasic/testprograms/TestPrograms.java

```diff
@@ -61,6 +61,9 @@ public void testPrograms() throws Exception {
         codeTest("TestEmpty.bas", "");
         codeTest("TestPrintHello.bas", "hello");
         codeTest("TestIf.bas", "111");
+        codeTest("TestSelect1.bas", "111111");
+        codeTest("TestSelect2.bas", "2468");
+        codeTest("TestSelect3.bas", "0 1 1 2 3 5 8");
```

Comment:
I would like to have test cases with syntax errors, nesting the different `select case` parts badly.

Reply:
I appended three test cases with incorrect syntax